sdk/pinocchio/src/sysvars/slot_hashes/test_std.rs

```diff
@@ -0,0 +1,113 @@
+//! Tests that rely on the `std` feature (host-only helpers, alloc, etc.).
+
+use super::*;
+use crate::{
+    account_info::{Account, AccountInfo},
+    pubkey::Pubkey,
+};
+use core::ptr;
+extern crate std;
+use super::test_utils::*;
+use std::io::Write;
+use std::vec::Vec;
+
+#[test]
+fn test_from_account_info_constructor() {
+    use std::eprintln;
+    eprintln!("DEBUG: Test starting");
+    std::io::stderr().flush().unwrap();
+
+    const NUM_ENTRIES: usize = 3;
+    const START_SLOT: u64 = 1234;
+
+    let mock_entries = generate_mock_entries(NUM_ENTRIES, START_SLOT, DecrementStrategy::Strictly1);
+    let data = create_mock_data(&mock_entries);
+
+    let mut aligned_backing: Vec<u64>;
+    #[allow(unused_assignments)]
+    let mut acct_ptr: *mut Account = core::ptr::null_mut();
```

Comment:
Why allow an unused assignment?
Rust compiler will make sure that you would not use an uninitialed value, right?

```suggestion
    let mut acct_ptr;
```